src/utils/generics.ts

```diff
@@ -185,6 +200,7 @@ export function mapGrantRewardAndListen(id: string, rewardJson: GrantReward): Re
 	const reward = new Reward(id)
 	reward.asset = rewardJson.asset
 	reward.committed = rewardJson.committed
+	if(rewardJson.token) reward.token = rewardJson.token.label
```

Comment:
This will just set the "token" of the reward to the "label" specified, won't link the token entity. I would recommend using  `mapWorkspaceTokens` here to save the token and reference the ID here